src/ops/view.rs

```diff
@@ -45,22 +48,22 @@ pub(super) fn pretty_view(
         (Some(latest), false) if latest.as_summary().version() != package_id.version() => {
             write!(
                 stdout,
-                " {warn}(latest {} {note}from {}{warn}){reset}",
+                " {warn}(latest {} {note}from {}{warn}){warn:#}",
```

Comment:
I am not sure if this is correct. Do we always need to reset first before we change color?

Reply:
Yes, we do.

Reply:
Thanks!

Reply:
You need to close each style before opening the next.

Reply:
Yes. I think I've updated it.